src/main/scala/fundamentals/level02/ListExercises.scala

```diff
@@ -85,6 +85,16 @@ object ListExercises {
     */
   def showListSize[A](xs: List[A]): String = ???
 
+  /**
+    * scala> val people = List(Person("Bob", 20), Person("Sally", 25))
+    * scala> sortByAgeDescending(people)
+    * = List(Person("Sally", 25), Person("Bob", 20))
+    *
+    * Hint: `sortBy` takes a function from `Person` to the field that you want to sort by.
```

Comment:
`sortBy` is defined as:

```
def sortBy[B](f: (A) ⇒ B)(implicit ord: math.Ordering[B]): Repr
```

So I wonder if we should mention that the field you want it sorted by should have an `Ordering` defined for it and then mention some common Orderings like Int, List, Float, Double, Boolean and String